python/test/test_polars.py

```diff
@@ -22,6 +30,31 @@ def test_infer():
     pl = pytest.importorskip("polars")
     series = pl.Series("A", [1] * 100)
     assert dp.RuntimeType.infer(series) == "Series"
+    assert dp.RuntimeType.infer(pl.DataFrame(series)) == "DataFrame"
+    assert dp.RuntimeType.infer(pl.LazyFrame(series)) == "LazyFrame"
+
+
+def example_lf(margin=None):
+    pl = pytest.importorskip("polars")
+    domains, series = example_series()
+    lf_domain, lf = dp.lazyframe_domain(domains), pl.LazyFrame(series)
+    if margin is not None:
+        lf_domain = _with_margin(lf_domain, margin)
+    return lf_domain, lf
+
+
+def _with_margin(domain, by):
+    return dp.with_margin(domain, by=by, public_info="lengths", max_partition_size=50)
+
+
+def test_lazyframe_domain_infer():
+    _, lf = example_lf()
+    assert dp.infer_lazyframe_domain(lf).member(lf)
+
+
+def test_domains():
+    with optional_dependency("polars"):
```

Comment:
`pytest.importskip` is probably preferable to this, because it will explicitly flag that the test was skipped, rather than giving us a pass when the test is a no-op.